enclave/src/lib.rs

```diff
@@ -47,6 +47,51 @@ fn check_chain_info(chain_info: *const u8, chain_info_len: usize) -> Option<Chai
     }
 }
 
+#[inline]
```

Comment:
Do we want to make this function inline?

Reply:
Yes, I think both are small utilities that would make sense to be inline -- they were extracted out to functions for reuse between two ecall functions (I'm still thinking whether to have multiple ecall functions or one) that unseal inputs and a bit better readability

Reply:
inline function makes code bigger, do you need inline for enclave compatibility?